adafruit_mcp4728.py

```diff
@@ -169,7 +178,7 @@ def save_settings(self):
         self._write_multi_eeprom(byte_list)
 
     # TODO: add the ability to set an offset
-    def _write_multi_eeprom(self, byte_list):
+    def _write_multi_eeprom(self, byte_list) -> None:
```

Comment:
My guess is that, based on the implementation, `byte_list` is intended to be `List[int]`.